runtime/compiler/p/codegen/J9CodeGenerator.cpp

```diff
@@ -123,6 +123,20 @@ J9::Power::CodeGenerator::initialize()
    if (!comp->getOption(TR_DisableReadMonitors))
       cg->setSupportsReadOnlyLocks();
 
+   if (!TR::Compiler->om.canGenerateArraylets()
+         && !TR::Compiler->om.isOffHeapAllocationEnabled()
+         && comp->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P8)
+         && comp->target().cpu.supportsFeature(OMR_FEATURE_PPC_HAS_VSX)
```

Comment:
are there anything in the fast-path code preventing it from being enabled for AOT (relocatable) or JITServer, as disabled below?